REFERENCE.md

```diff
@@ -549,6 +550,14 @@ Minimum number of slaves to be in "online" state
 
 Default value: `0`
 
+##### <a name="modules"></a>`modules`
+
+Data type: `Array[Stdlib::Absolutepath]`
+
+Absolute path of additional modules to load
```

Comment:
This string seems outdated comgared to the changes bellow.  Do not bother updating `REFERENCE.md` since it can easily introduce conflicts with other contributors: it will be updated during the release process :wink:

Reply:
Ok, I will remove this.